packages/wire/lib/messages/ChannelUpdateMessage.ts

```diff
@@ -31,13 +31,13 @@ export class ChannelUpdateMessage implements IWireMessage {
         instance.messageFlags = BitField.fromNumber(reader.readUInt8());
         instance.channelFlags = BitField.fromNumber(reader.readUInt8());
         instance.cltvExpiryDelta = reader.readUInt16BE();
-        instance.htlcMinimumMsat = reader.readUInt64BE();
+        instance.htlcMinimumMsat = Value.fromMilliSats(reader.readUInt64BE());
         instance.feeBaseMsat = reader.readUInt32BE();
```

Comment:
`feeBaseMsat` should probably be a `Value` as well.  Any thoughts on `feeProportionalMillionths`?

Reply:
* I tried `feeBaseMsat` with type `Value`  it worked for deser but in case of ser --> I got an error that bigInt is not assignable to number. This is because while writing the value to buffer in Value.ts it isn't compatible with numbers. I can either allow the number access as shown below, 
![image](https://user-images.githubusercontent.com/40585900/128745741-93a9eb51-e7e6-4a25-b6d3-eae0084aa34d.png)<br>
or 
![image](https://user-images.githubusercontent.com/40585900/128746208-ccf31107-2cce-4e50-9bb7-a8e0af31d1bd.png)
In this I am making it `bigint` type.

*  `feeProportionalMillionths`  is like 1/1e6 of a satoshi, similar to 
![image](https://user-images.githubusercontent.com/40585900/128746612-e19b2dd4-5951-489d-ad87-fcd25927434b.png)
 If you want I can introduce a corresponding fxn for it in Value.ts file.

Reply:
Sure that sounds like a great idea!